helpers/safe-fetch.ts

```diff
@@ -23,7 +20,7 @@ export async function safeFetch(
     return {
         headers: response.headers,
         status: response.status,
-        text: async () => {
+        text: () => {
```

Comment:
This is wrong. See the failed text for more information.

Reply:
fixed